test/activities/quizzes/data/submissionViews/SubmissionViewsEntity.js

```diff
@@ -0,0 +1,58 @@
+export const editableCollection = {
+	class: ['submission-views', 'collection'],
+	entities: [
+		{
+			class: ['submission-view', 'primary'],
+			rel: ['item'],
+			href:
+				'/{orgUnitId}/quizzes/{quizId}/submissionviews/7?workingCopyId=123'
+		},
+		{
+			class: ['submission-view', 'secondary'],
+			rel: ['item'],
+			href:
+				'/{orgUnitId}/quizzes/{quizId}/submissionviews/9?workingCopyId=123'
+		}
+	],
+	links: [
+		{
+			rel: ['self'],
+			href:
+				'/{orgUnitId}/quizzes/{quizId}/submissionviews?workingCopyId=123'
+		}
+	],
+	actions: [
+		{
+			href:
+				'/{orgUnitId}/quizzes/{quizId}/submissionviews?workingCopyId=123',
+			name: 'add',
+			method: 'POST',
+			fields: []
+		}
+	]
+};
+
+export const nonEditableCollection = {
+	class: ['submission-views', 'collection'],
+	entities: [
+		{
+			class: ['submission-view', 'primary'],
+			rel: ['item'],
+			href:
+				'/{orgUnitId}/quizzes/{quizId}/submissionviews/7?workingCopyId=123'
+		},
+		{
+			class: ['submission-view', 'secondary'],
+			rel: ['item'],
+			href:
+				'/{orgUnitId}/quizzes/{quizId}/submissionviews/9?workingCopyId=123'
+		}
+	],
+	links: [
```

Comment:
Thinking back to the work I did in siren serializer...🤔  Would `actions` just not exist or show up as an empty array?

Reply:
Don't know, but don't think it matters.
If it breaks, then `hasActionByName` has a bug